common-content/en/module/piscine/practice-break-down/index.md

```diff
@@ -0,0 +1,24 @@
++++
+title = "Practice breaking down a requirement"
+time = 60
+objectives = [
+  "Break down a requirement into approachable tasks.",
+  "Coordinate multiple team members in executing tasks in parallel.",
+]
+[build]
+  render = "never"
+  list = "local"
+  publishResources = false
++++
+
+In team projects, it's important that we break large tasks down into small tasks.
+
+It's also important that we can coordinate across our team. This requires having shared understanding of who will do what, and how the work we do will interact.
+
+We will practice this together on the first two requirements of the project.
```

Comment:
Did we want to document the broken down tickets here?

Reply:
Works for me! I think you signed up to do the break-down, so shall we merge this and your can PR to it?

Also thinking about the break-down, it occurs to me that the first two tickets don't involve much cross-team coordination - do we maybe want to include in our break down something that e.g. requires agreeing on a serialisation format? Or would that be too much solving implementation things for them?

Reply:
Yeah, I realised that requirement 2 is most of the rendering aspect of the project, plus the de-serialisation.

It's actually surprisingly tricky - I think I've also come around to the conclusion that thinking about the serialisation is probably the best topic to tackle here.

Reply:
> so shall we merge this and your can PR to it?

Yep sounds good!

Reply:
Sounds good :) will merge for now and look forward to your PR!